cmd/ddev/cmd/delete-images.go

```diff
@@ -1,26 +1,32 @@
 package cmd
 
 import (
+	"os"
+	"sort"
+	"strings"
+
 	"github.com/drud/ddev/pkg/dockerutil"
 	"github.com/drud/ddev/pkg/nodeps"
 	"github.com/drud/ddev/pkg/util"
 	"github.com/drud/ddev/pkg/version"
 	docker "github.com/fsouza/go-dockerclient"
 	"github.com/spf13/cobra"
-	"os"
-	"sort"
-	"strings"
 )
 
 // noConfirm: If true, --yes, we won't stop and prompt before each deletion
 var deleteImagesNocConfirm bool
 
+// deleteAllImages: If set, deletes all images created by ddev
+var deleteAllImages bool
+
 // DeleteImagesCmd implements the ddev delete images command
 var DeleteImagesCmd = &cobra.Command{
 	Use:   "images",
-	Short: "Delete docker images not currently in use",
+	Short: "Deletes ddev docker images",
```

Comment:
```suggestion
	Short: "Deletes ddev docker images not in use by current ddev version",
	Long: "with --all it deletes all ddev docker images"
```

Reply:
I will make your suggested change.

Reply:
All you have to do for something like this is click the "Commit suggestion" button below it.